plugins/webhooks/index.js

```diff
@@ -86,13 +152,15 @@ function startPRJob(options, request) {
             prInfo,
             prRef,
             prNum,
-            startFrom: '~pr',
+            startFrom,
```

Comment:
Do we need `configPipelineSha` for restarting in events?
Reference: https://github.com/screwdriver-cd/screwdriver/pull/1165/files